wordpress-mobile/WordPress-Android
# Issue/4571 simperium lag on comment pn open

Fixes #4571 

This implements the following solution as explained in the referenced issue:

```
c) we try fetching the note detail from the REST api, but for this we need to know the remote blog id and comment id, which we should be sending along with the PN payload (we aren’t doing so right now).
```

To test: 

In order to test this, it is first necessary that the server sends `blog_id` and `comment_id` as keys in the PN payload. 
(There's a diff patch for that waiting for review)

1. comment out the contents of method `startBuckets()`  in file `SimperiumUtils.java`. This will prevent Simperium from actually trying to sync buckets, so we make sure we are not getting info from Simperium as per the purpose of this test.
2. generate a comment on a blog you're an admin of (make the comment with another test user).
3. you should receive a push notification  (which internally carries `blog_id` and `comment_id` ids)
4. tap on such notification
5. the app will fetch the information through WP REST API and show the comment contents to the user.
6. also, the user should be able to moderate the comment accordingly

Needs review: @daniloercoli
